docs/resources/catalog.md

```diff
@@ -27,7 +27,7 @@ resource "databricks_catalog" "sandbox" {
 The following arguments are required:
 
 * `name` - Name of Catalog relative to parent metastore. Change forces creation of a new resource.
-* `owner` - (Optional) Username/groupname of catalog owner. Currently this field can only be changed after the resource is created.
+* `owner` - (Optional) Username/groupname/sp id of catalog owner.
```

Comment:
make it a bit more explicit, is it `application_id` or `id` from `databricks_service_principal`? :) please be more explicit